code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -116,13 +119,14 @@
 
 /datum/reagent/toxin/potassium_chloride/overdose(var/mob/living/carbon/M, var/alien)
 	..()
-	if(ishuman(M))
-		var/mob/living/carbon/human/H = M
-		if(H.stat != 1)
-			if(H.losebreath >= 10)
-				H.losebreath = max(10, H.losebreath - 10)
-			H.adjustOxyLoss(2)
-			H.Weaken(10)
+	var/mob/living/carbon/human/H = M
+	if(istype(H) && (H.species.flags & NO_BLOOD))
```

Comment:
This is not the logical equivalent of what was before. The lower if will still active if mob is **not** human, which is bad.

Use this one instead: `if (!istype(H) || (H.species.flags & NO_BLOOD))`

It'll stop if the mob is not human OR if the mob is human but has NO_BLOOD.